crates/gpio/Cargo.toml

```diff
@@ -9,7 +11,10 @@ keywords = ["gpio", "vhost", "virt", "backend"]
 license = "Apache-2.0 OR BSD-3-Clause"
 edition = "2021"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[features]
+default = ["dummy"]
```

Comment:
I am not in favor of this being supported by default. This is only for special purposes, like testing, etc. Even if the size change in binary is pretty small, we shouldn't enable it by default.

And feature name should be something like mock-gpio, etc. I already commented earlier to change the name "dummy", as it doesnt' fit well anymore.

Reply:
You also need to enable it when tests are enabled.